src/components/organisms/body-card.tsx

```diff
@@ -64,11 +64,15 @@ const BodyCard: React.FC<BodyCardProps> = ({
 
           <div className="flex items-center space-x-2">
             {status && status}
-            <Actionables
-              actions={actionables}
-              forceDropdown={forceDropdown}
-              customTrigger={customActionable}
-            />
+            {
+              actionables ? (
+                <Actionables
+                  actions={actionables}
+                  forceDropdown={forceDropdown}
+                  customTrigger={customActionable}
+                />
+              ) : null
+            }
```

Comment:
Please undo this change, as it breaks the UI in several places 😄 
```suggestion
  <Actionables
              actions={actionables}
              forceDropdown={forceDropdown}
              customTrigger={customActionable}
            />
```